dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.js

```diff
@@ -128,19 +128,19 @@ const findAmendmentsByDealId = async (dealId) => {
   try {
     const collection = await db.getCollection('tfm-facilities');
     const amendment = await collection
-      .aggregate([ // TODO SR-8
-        { $match: { 'facilitySnapshot.dealId': ObjectId(dealId) } },
+      .aggregate([
+        { $match: { 'facilitySnapshot.dealId': { $eq: ObjectId(dealId) } } },
         {
           $addFields: {
             'amendments.ukefFacilityId': '$facilitySnapshot.ukefFacilityId',
           },
         },
-        { $project: { _id: 0, amendments: 1 } },
+        { $project: { _id: false, amendments: true } }, // TODO SR-8 Changed values in project to true or false to better represent the functionality
```

Comment:
Remove todo once noted:
```suggestion
        { $project: { _id: false, amendments: true } },
```